### ISSUE TITLE
Export Query List

### ISSUE THREAD

**Issue by goodhotdogs** [AdguardTeam/AdGuardHome#3389]
We
- [Yes] I am running the latest version
- [Yes ] I checked the documentation and found no answer
- Yes[ ] I checked to make sure that this issue has not already been filed

Being Able to Export the Query List would be a great feature i believe it has been before, i don't know why it was removed.

Thanks

**Comment by ainar-g** [comment]
Hello.  What exactly do you mean by “Query List”?  Do you mean the query log?

**Comment by goodhotdogs** [comment]
> Hello. What exactly do you mean by “Query List”? Do you mean the query log?

Yes, the query log it would be great if we could download the query log

**Comment by ainar-g** [comment]
I see, thanks.  Downloading the whole query log may be problematic, since it may be up to several GiB long.  And that will probably have to wait until #2290 is decided and done.

**Comment by Eyeborgs** [comment]
Would also like to have a log export for individual clients

**Comment by goodhotdogs** [comment]
Yes, that would also be great.

**Comment by goodhotdogs** [comment]
Even just the log as a txt file

**Comment by jcdescy** [comment]
Still open and I would still like to have this.
